examples/views/dropdown.py

```diff
@@ -0,0 +1,38 @@
+import discord
+from discord.ext import commands
+
+class Bot(commands.Bot):
+    def __init__(self):
+        super().__init__(command_prefix=commands.when_mentioned_or('$'))
+
+    async def on_ready(self):
+        print(f'Logged in as {self.user} (ID: {self.user.id})')
+        print('------')
+
+#Gives us a view containing a dropdown menu
+class Dropdown(discord.ui.View):
+    def __init__(self):
```

Comment:
```suggestion
    def __init__(self, *, timeout: typing.Optional[float] = 180):
```

Reply:
This `__init__` should either not be overriden or match the signature of the superclass.